reana_workflow_controller/rest/workflows.py

```diff
@@ -180,10 +197,24 @@ def get_workflows(paginate=None):  # noqa
         type_ = request.args.get("type", "batch")
         verbose = json.loads(request.args.get("verbose", "false").lower())
         block_size = request.args.get("block_size")
+        sort = request.args.get("sort", "desc")
+        search = request.args.get("search", "")
+        status_list = request.args.get("status", "")
```

Comment:
Just wanted to be double safe in case some other service would call this endpoint. In that case (without the second argument) `status_list` would be `None` and there would be some troubles to `.split(",")` it.

Reply:
But if it's `None` it won't pass the check on line 209, so `split` won't happen.
Anyway, fair enough, it's not very important and it's always good to double-check 🔒